examples/example-gateway/idl/endpoints/contacts/contacts.thrift

```diff
@@ -30,7 +30,7 @@ struct Contact {
 }
 
 struct SaveContactsRequest {
-    1: required UUID userUUID (zanzibar.http.ref = "params.userUUID");
+    1: required string userUUID (zanzibar.http.ref = "params.userUUID");
```

Comment:
curious why

Reply:
if it's uuid then we have to generate code like `args.Request.userUUID = UUID(req.Params.ByName('userUUID'))`

Generating that convert a string to `UUID` alias logic is non-trivial.